### PR TITLE
feat(#1623): refactor `OyFallbackTest`

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
1. Add `OyMock` for easy testing
2. Add tests for all `OyFallback` methods 

Closes: #1623

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@mximp Can you take a look, please?

File: eo-maven-plugin/src/test/java/org/eolang/maven/objectionary/OyFallbackTest.java [hunk 1]
```diff
@@ -23,72 +23,148 @@
...
+        private OyMock(final String object) {
+            this.name = object;
+            this.invocations = new AtomicInteger();
+        }
+
+        @Override
+        public Input get(final String object) throws IOException {
+            this.invocations.incrementAndGet();
+            if (this.name.equals(object)) {
+                return new InputOf(OyFallbackTest.SOURCE);
```
Comment (mximp) [inline eo-maven-plugin/src/test/java/org/eolang/maven/objectionary/OyFallbackTest.java:158]: @volodya-lombrozo I believe it's better to make the `source` internal to `OyMock`. Let's pass the source as another constructor param and eliminate dependency on `OyFallbackTest`

Comment (volodya-lombrozo) [reply eo-maven-plugin/src/test/java/org/eolang/maven/objectionary/OyFallbackTest.java]: @mximp Indeed, It makes sense. I've changed the code a little bit. Can you take a look again, please?

**Review by mximp — APPROVED** [review]

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**